code/datums/helper_datums/hotkey_modes.dm

```diff
@@ -2,6 +2,14 @@
 	var/name
 	var/macro_hotkeys_inactive
 	var/macro_hotkeys_active
+	var/mob/my_mob
+
+/datum/hotkey_mode/New(mob)
+	my_mob = mob
+
+/datum/hotkey_mode/proc/set_winset_values()
+	winset(my_mob, null, "mainwindow.macro=[macro_hotkeys_inactive] hotkey_toggle.is-checked=false input.focus=true input.background-color=#d3b5b5")
+	winset(my_mob, null, "hotkey_toggle.command=\".winset \\\"mainwindow.macro != [macro_hotkeys_inactive] ? mainwindow.macro=[macro_hotkeys_inactive] hotkey_toggle.is-checked=false input.focus=true input.background-color=#d3b5b5 : mainwindow.macro=[macro_hotkeys_active] hotkey_toggle.is-checked=true mapwindow.map.focus=true input.background-color=#f0f0f0\\\"\"")
```

Comment:
If you won't ask, I won't tell. These escaped quotes and slashes is the only way to get this to work.

Reply:
It makes sense, you'd need to do the same for declaring attributes of an HTML tag, for example.